<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Charge_settings_regist</class>
 <widget class="QMainWindow" name="Charge_settings_regist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>樣板</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>掛號費設定</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QToolButton" name="toolButton_regist_fee_add">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../icons/list-add.svg</normaloff>../icons/list-add.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_regist_fee_delete">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../icons/list-remove.svg</normaloff>../icons/list-remove.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_regist_fee_edit">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../icons/gtk-edit.svg</normaloff>../icons/gtk-edit.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget_regist_fee">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <column>
              <property name="text">
               <string>ChargeSettingsKey</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>收費類別</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>項目名稱</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>門診類別</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>負擔類別</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>就醫類別</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>療程</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>收費金額</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>備註</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>掛號費及部份負擔優待設定</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QToolButton" name="toolButton_discount_add">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../icons/add.svg</normaloff>../icons/add.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_discount_delete">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../icons/gtk-remove.svg</normaloff>../icons/gtk-remove.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_discount_edit">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../icons/gtk-edit.svg</normaloff>../icons/gtk-edit.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget_discount">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <column>
              <property name="text">
               <string>ChargeSettingsKey</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>收費類別</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>優待名稱</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>優待金額</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QCheckBox" name="checkBox_old_man">
               <property name="text">
                <string>年長病患掛號費自動優待</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>年長年齡定義</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_old_man_age">
               <property name="minimum">
                <number>60</number>
               </property>
               <property name="maximum">
                <number>120</number>
               </property>
               <property name="value">
                <number>65</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>歲以上</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;
&lt;body&gt;
優待金額設定方式:
&lt;ul&gt;
&lt;li&gt;
設定正數, 代表掛號費收此金額 
&lt;/li&gt;
&lt;li&gt;
設定負數, 代表掛號費減免此金額. 
&lt;/li&gt;
&lt;/ul&gt;

&lt;/body&gt;
&lt;/html&gt;</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
